Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,63 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+# Tips and Tricks
+
+This is a list of useful tips and tricks to help you make the most out of SerenityOS
+
+Related: [Keyboard Shortcuts](help://man/7/KeyboardShortcuts)
+
+## General
+* When on the Desktop or in File Manager, start typing and the nearest match will be selected
+* Bold text in context menus hints at the default behavior of a double-click
+* Hold `Ctrl` to accelerate mouse wheel interaction with sliders and spin boxes
+* Hold `Ctrl` while activating a menu item to prevent that menu from closing
+* Press `Ctrl+Shift+A` on a focused widget or application to show the Command Palette, a searchable list of available actions
+* Many applications will open a compatible file if you drag-and-drop it into their window
+* Default file and protocol associations can be changed in `~/.config/LaunchServer.ini`
+
+### Window Management
+* Double-click a window's title bar to maximize it
+* Click on a window's icon to open the context menu
+* Double-click on the edge of an application's window to maximize it in that direction
+* Middle-click on a window's maximize button to extend the window vertically (this can be undone the same way)
+* Resizable windows can be snapped to all edges of the screen. Drag a window to an edge or press `Super+Left, Right or Up` while it has focus
+* Cycle focus between windows by pressing and holding `Super+Tab`
+    * Use `Shift+Super+Tab` to do this in the reverse order
```

Comment:
That should be `Alt+Tab`, and it doesn't belong here.

Reply:
Do Alt+Tab and Super+Tab do the same thing?

I can't test Super+Tab on macOS as its already a system shortcut (with no obvious way to switch off).
This information was taken from the original tips.txt and Xexxa's Wiki.

Reply:
Alt+Tab has the same function as in any other operating system, which is the exact function you're describing above. There are virtualization problems with it on (as you say) MacOS and (as I know) Linux, (as I know) it works better on Windows. I think documenting the normal shortcut is a better idea since this function is what every user will expect.